src/content/ccip/concepts.mdx

```diff
@@ -31,7 +31,7 @@ To learn more, read about [blockchain interoperability](https://chain.link/educa
 
 ## Finality
 
-**Finality** refers to the state of irreversibility and permanent record of a transaction on the blockchain. If the parameters for finality are properly set, the likelihood of irreversibility is extremely low. For CCIP, source chain finality is the main factor that determines the end-to-end elapsed time for CCIP to send a message from one chain to another.
+**Finality** is the assurance that past transactions included on-chain are extremely difficult, if not impossible, to revert. If the parameters for finality are properly set, the likelihood of irreversibility is extremely low. For CCIP, source chain finality is the main factor that determines the end-to-end elapsed time for CCIP to send a message from one chain to another.
```

Comment:
> the likelihood of irreversibility is extremely low.

should this say "the likelihood of _reversibility_ is extremely low"

Reply:
Fixed